docs/fedora-releases-guide/what-to-know.md

```diff
@@ -0,0 +1,112 @@
+
+:::
+
+:::warning Packit pull requests can spam other maintainers
+
+SPam -> ask more about the spam
+
+:::
+
+:::warning missing notifications
+
+Packit bot user creates pull requests, Koji builds, and Bodhi updates, making it the default notification recipient.
+
+To receive pull request comment notifications, configure notification settings accordingly.
+
+There is an [open issue](https://github.com/packit/packit-service/issues/2404) for Bodhi update notifications.
+
+:::
+
+## Monitoring and observability
+
+Track *Packit Service* activity through:
+- the pull request updates and 
+- the [Packit Dashboard](https://dashboard.packit.dev/).
+
+Packit Service responds with 👀 for every valid command in pull requests. Missing eyes indicate:
+- Incorrect command syntax
+- Packit Service outage
+
+Check service status at [https://status.packit.dev/](https://status.packit.dev/).
+
+## Recovery procedures
+
+### Service-based retriggering
+
+Each job configured in `packit.yml` can be retriggered via dist-git pull request comments. See the [complete list of commands](https://packit.dev/docs/fedora-releases-guide/dist-git-onboarding#retriggering).
+
+### Manual CLI Operations
+
+During service outages, maintain workflow using Packit CLI:
+
+1. Install [Packit CLI](https://packit.dev/docs/cli)
+2. [Configure required tokens](https://packit.dev/docs/configuration#user-configuration-file)
+3. Verify setup:
+   - Follow [dist-git onboarding guide](https://packit.dev/docs/fedora-releases-guide/dist-git-onboarding)
+   - Validate config with `packit validate-config`
+
+#### Troubleshooting Common Issues
+
+**Missing pull request for release:**
+
+Use [`packit pull_from_upstream`](https://packit.dev/docs/cli/pull-from-upstream) when [release-monitoring](https://release-monitoring.org/) detects updates but pull requests aren't created.
+
+**Missing Koji build:**
+
+The release pull request is merged (and contains a change in the version value) but a Koji build isn't triggered.
+
+- Check [`allowed_pr_authors`](https://packit.dev/docs/configuration/downstream/koji_build#optional-parameters)
+- Trigger build manually with [`packit build in-koji`](https://packit.dev/docs/cli/build/in-koji)
+
+When using Packit CLI to create pull requests, ensure your FAS account is added to the `allowed_pr_authors` list to trigger Koji builds automatically. This enables the expected build workflow for maintainer-created pull requests.
+
+**Missing Bodhi update:**
+
+The Koji build is created but there is no Bodhi update.
+
+- Packit does not create Bodhi updates for `rawhide`, they are automatically created by Bodhi.
+- Verify [`allowed_builders`](https://packit.dev/docs/configuration/downstream/bodhi_update#optional-parameters)
+- Create manually using [`packit create-update`](https://packit.dev/docs/cli/create-update)
```

Comment:
Why are CLI commands suggested before checking the dashboard for erros and retriggering (that's not even mentioned)?

Reply:
Mmmm both checking the dashboard and retriggering are above the CLI usage 🤔 not sure if I follow.

Reply:
Oh, sorry, I didn't realize that `Troubleshooting Common Issues` is a CLI subsection.